scripts/west_commands/boards.py

```diff
@@ -49,8 +49,9 @@ def do_add_parser(self, parser_adder):
             The following arguments are available:
 
             - name: board name
-            - identifiers: board identifiers
-            - arch: board architecture
+            - identifiers: board identifiers (will be empty for legacy boards)
```

Comment:
we do not call this identifiers anymore, no?

Another thing, `west boards -f "{identifiers}"`  gives me a list of qualifiers, if a board has multiple, they are joined by a comma, so if I want to get the list of targets using "-f", it becomes very difficult. How about adding `{targets}`

Reply:
ok, read the comment above re qualifiers, but rest of the above still applies.

Reply:
> so if I want to get the list of targets using "-f", it becomes very difficult. How about adding {targets}

yes, but in this initial PR i didn't want to trigger this discussion again related to legacy vs new model boards:
https://github.com/zephyrproject-rtos/zephyr/pull/69740#discussion_r1522772483

Therefore this initial PR which simply ensures that issues are fixed and completion works.
Then a second PR allowing for longer discussion to take place.

I don't like to make an unnecessary block on the fix in this PR.